.github/workflows/tests.yml

```diff
@@ -15,19 +15,31 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["2.7", "3.10", "pypy-2.7", "pypy-3.7"]
+        python-version:
+          - "2.7"
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "pypy-2.7"
+          - "pypy-3.7"
+          - "pypy-3.9"
```

Comment:
I figured this was a large enough change to justify more thorough testing. Happy to revert this prior to merge.

Reply:
yeah, let's not bother with 3.8 3.9 3.10 in this case.